libs/platform/src/lib/form/form-generator/base-dynamic-form-generator-control.ts

```diff
@@ -35,10 +36,31 @@ export abstract class BaseDynamicFormGeneratorControl implements BaseDynamicForm
     @Input() form: FormGroup;
 
     /**
-     * @description Reference to the @see FormFieldComponent
+     * @description Reference to the @see FormFieldComponent.
      */
     @Input() formField: FormField;
 
     /** @description Inner form group name */
     @Input() formGroupName: string;
+
+    /** @description Returns form item content density. */
+    get contentDensity(): ContentDensity | undefined {
+        return this.formItem.guiOptions?.contentDensity || undefined;
```

Comment:
nullish coalescing gives `undefined` by itself
```suggestion
        return this.formItem.guiOptions?.contentDensity;
```

Reply:
@dmitry-stepanenko it gave `null`, because of that there was an issue with content density for form